src/Microsoft.ML.Transforms/Text/doc.xml

```diff
@@ -221,7 +226,14 @@
     <example name="WordEmbeddings">
       <example>
         <code language="csharp">
-          pipeline.Add(new WordEmbeddings((&quot;InVectorTextCol&quot; , &quot;OutTextCol&quot;)));
+            pipeline.Add(new TextFeaturizer(&quot;Words&quot;, &quot;InputTextCol&quot;)
+            {
+                TextCase = TextNormalizerTransformCaseNormalizationMode.Lower,
+                OutputTokens = true,
```

Comment:
I think it looks bit odd here to use TextFeaturizer for tokenization only. Wouldn't it be nice to use simply wordtokenizer in this example.

Reply:
Yes, it's weird, but Wordtokenizer doesn't do lower casing for words and AFAIK all models use lowercased words.

Reply:
We have TextNormalizer for that but I don't find it in CSharpApi.cs. Is it exposed in public API?

Reply:
Using the TextFeaturizer is the correct thing to do. As @Ivanidzo4ka says, we need the lower casing, punctuation removal, and (sometimes) the stop word removal.

And in most cases you want to get the ngrams at the same time, as this is the winning method (wordembeddings + ngrams + chargrams).